prow/jobs/images/Dockerfile.deploy

```diff
@@ -9,6 +9,7 @@ RUN dnf -y install \
     && curl -sL https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip -o awscliv2.zip \
     && unzip awscliv2.zip \
     && aws/install \
+    && export AWS_PAGER="" \
```

Comment:
what's this?

Reply:
This makes sure aws-cli output does not open with `less` but redirected to stdout instead.
Copied it from Nick's work on e2e tests.

Reply:
You can't do `variable=$(aws ...)` unless you set this variable, because any `aws` output is redirected to `less`. It's an interesting choice.